firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -111,7 +112,7 @@ bool moveStraight(int motor[], int startCol, int startRow, int endCol, int endRo
     return false;
   }
 
-  numSteps = spaces * stepsPerSpace;
+  numSteps = 2 * pieceSpaces * stepsPerSpace;
```

Comment:
This will give you an error if you compile the code. Since pieceSpaces is a float.